packages/edit-site/src/components/global-styles/screen-block-list.js

```diff
@@ -19,6 +20,29 @@ import { useHasTypographyPanel } from './typography-panel';
 import ScreenHeader from './header';
 import { NavigationButton } from './navigation-button';
 
+function useSortedBlocks() {
```

Comment:
Also a renaming to `useSortedBlockTypes` could be better, as we are handling `block types` and not blocks.